qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -450,33 +450,26 @@ def _submit_callback(self, job_name=None):
         backend_name = self.backend().name()
 
         submit_info = None
-        if self._use_object_storage:
-            # Attempt to use object storage.
-            try:
+        try:
+            if self._use_object_storage:
+                # Submit Qobj via object storage.
```

Comment:
Small nit - can yo use ` Submit Job ...` instead of `Qobj`, for clarity? (also on the else)